examples/ansible/setup_keylime.yml

```
@@ -0,0 +1,49 @@
+---
+# Ansible Playbook: Setup Keylime on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment and setup of Keylime verifier, registrar and tenant on a SUSE ALP Dolomite host
+# The steps encompass fetching the Keylime image from the specified registry, creating a necessary volume for the Keylime control plane,
+# launching the Keylime container along with related services, and executing the tenant CLI to interact with Keylime.
+# Documentation reference: [https://documentation.suse.com/alp/dolomite/html/alp-dolomite/keylime-remote-attestation.html#keylime-run-with-podman]
+
+- name: Setup Keylime verifier, registrar and tenant
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: keylime
+      image: registry.opensuse.org/devel/microos/containers/containerfile/opensuse/keylime-control-plane:latest
+
+  tasks:
+    - name: Retrieve images for Keylime
+      containers.podman.podman_image:
+        name: "{{ workload.image }}"
+        state: present
+
+    - name: Create the keylime-control-plane volume
+      containers.podman.podman_volume:
+        name: keylime-control-plane-volume
+        state: present
+
+    - name: Start the Keylime container and related services
+      ansible.builtin.command: >-
+        podman container runlabel run "{{ workload.image }}"
+      register: workload_runlabel_run
+      changed_when:
+        - workload_runlabel_run.stdout is regex("^[a-f0-9]{64}$")
+
+    - name: Pause for 5 seconds
+      ansible.builtin.pause:
+        seconds: 5
+
+    - name: Executing the tenant CLI
+      ansible.builtin.command: >-
+        podman run --rm -v keylime-control-plane-volume:/var/lib/keylime/ keylime-control-plane:latest
+        keylime_tenant -v 10.88.0.1 -r 10.88.0.1 --cert default -c reglist
+      register: podman_run
+      changed_when:
+        - "'\"status\": \"Success\"' in (podman_run.stdout_lines | join(' '))"
```

You should be able to avoid the need for the awkward quoting here by using parentheses rather than quotes, e.g.

```suggestion
        - ('"status": "Success"' in (podman_run.stdout_lines | join(' ')))
```

---

IIRC the Ansible parser is unhappy if the first character of a when clause is a quote and the last character is not the matching close quote for the entire string. However, parenthesizing the when clause avoids the first character being a quote, avoiding the need for awkward quoting if you need to use both types of quoting within the when clause itself.

---

Also note that if the output of the command is a "pure" JSON blob with no extraneous characters, you could use the `from_json` filter to convert the value into a Python dict, and access the `status` element appropriately to check it's value...

---

The output is not pure JSON & facing single-quoted scalar issue with -
  ('"status": "Success"' in (podman_run.stdout_lines | join(' ')))
 using-
  (podman_run.stdout | regex_search('"status":\\s+"Success"')) is not none